_pkgdown.yml

```diff
@@ -107,6 +107,7 @@ reference:
       - read_cmdstan_csv
       - write_stan_json
       - write_stan_file
+      - draws_to_csv
```

Comment:
I added this so it will be listed in the section with these other functions on the website

Reply:
Ah, nice! missed that.